cnf-tests/docgen/e2e.json

```diff
@@ -33,6 +35,7 @@
     "[rfe_id:27368][performance] Pre boot tuning adjusted by tuned  [test_id:31198] Should set CPU affinity kernel argument": "Checks that the node has injected systemd.cpu_affinity argument under boot parameters, that used to configure the CPU affinity",
     "[rfe_id:27368][performance] Pre boot tuning adjusted by tuned  [test_id:32375][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] initramfs should not have injected configuration": "Checks that the iniramfs does not have injected configuration",
     "[rfe_id:27368][performance] Pre boot tuning adjusted by tuned  [test_id:35363][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running on the host": "Checks that the stalld daemon is running on the host",
+    "[rfe_id:27368][performance] Pre boot tuning adjusted by tuned  [test_id:42400][crit:medium][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running as sched_fifo": "Checks that the stalld daemon is running as sched_fifo",
```

Comment:
where is this coming from ?

Reply:
Automatically generated after I ran ```make check-tests-nodesc``` and ```make generate-cnf-tests-doc```